ui/components/app/name/name-details/name-details.tsx

```diff
@@ -383,7 +383,12 @@ export default function NameDetails({
             <div
               style={{ textAlign: 'center', marginBottom: 16, marginTop: 8 }}
             >
-              <NameDisplay value={value} type={type} variation={variation} />
+              <NameDisplay
+                value={value}
+                type={type}
+                variation={variation}
+                showFullName
```

Comment:
[nit]: this is more personal than anything but I feel that being explicit on the props value increases readability. So my suggestion is to use `showFullName={true}` instead of just `showFullName`.